client/test/accounts_lib.spec.js

```diff
@@ -0,0 +1,95 @@
+'use strict'
+
+const assert = require('chai').assert
+
+const AccountsLib = require('../app/libs/accounts')
+
+
+describe("Accounts libs spec", () => {
+
+    describe("Validate and returns a new state", () => {
+
+        it("should returns a simple value in a new state", () => {
+            let state = {}
+            let nextState = AccountsLib.validateState('login', 'mail@cozy.io', state)
```

Comment:
That's what I thought, I was not sure I would have kept with consistency in this case. I wondered if there was something else. Thanks!